opm/material/common/Tabulated1DFunction.hpp

```diff
@@ -421,7 +426,6 @@ class Tabulated1DFunction
                yValues_ == data.yValues_;
     }
 
-private:
     template <class Evaluation>
     size_t findSegmentIndex_(const Evaluation& x, bool extrapolate = false) const
```

Comment:
Should rename this method to lose the underscore, since it is no longer private.

Reply:
done